docs/source/howto/installation.rst

```diff
@@ -293,68 +293,57 @@ This can be useful to temporarily enable deprecation warnings for a single comma
 
-.. code:: bash
+1. First, the ``AIIDA_PATH`` environment variables is checked, which can be a colon-separated list of directories.
+The first directory that points to an existing configuration directory is selected.
+If no existing directories are found, the last directory defined in the variable is used.
+2. If the ``AIIDA_PATH`` is not defined, the current working directory is checked, going up the hierarchy until the first existing configuration directory is encountered.
+3. If no existing configuration directory is found yet, the ``.aiida`` directory in the user's home folder is used, and is created automatically if it does not already exist.
 
-    Info: configuration folder: /home/user/.virtualenvs/aiida/.aiida
-    Critical: configuration file /home/user/.virtualenvs/aiida/.aiida/config.json does not exist
-
-The second line you will only see if you haven't yet setup a profile for this AiiDA instance.
-For information on setting up a profile, refer to :ref:`creating profiles<how-to:installation:profile>`.
+Examples
+........
 
-Besides a single path, the value of ``AIIDA_PATH`` can also be a colon-separated list of paths.
-AiiDA will go through each of the paths and check whether they contain a configuration directory, i.e., a folder with the name ``.aiida``.
-The first configuration directory that is encountered will be used as the configuration directory.
-If no configuration directory is found, one will be created in the last path that was considered.
-For example, the directory structure in your home folder ``~/`` might look like this::
+Consider the following directory structure::
 
-    .
+    ~
     ├── .aiida
-    └── project_a
-        ├── .aiida
+    ├── project_a
+    │   ├── .aiida
+    │   └── subfolder
+    │       └── .aiida
+    ├── project_b
+    │   ├── .aiida
+    │   └── subfolder
+    └── project_c
         └── subfolder
 
-If you leave the ``AIIDA_PATH`` variable unset, the default location ``~/.aiida`` will be used.
-However, if you set:
-
-.. code:: bash
-
-    $ export AIIDA_PATH='~/project_a:'
+The following table shows the configuration directory that is selected given a certain ``AIIDA_PATH`` variable and the current working directory:
 
-the configuration directory ``~/project_a/.aiida`` will be used.
++--------------------------------------------------------------+----------------------------------+
+| Variables                                                    | Configuration directory          |
++==============================================================+==================================+
+| ``AIIDA_PATH = '~/project_b/'``                              | ``~/project_b/.aiida``           |
+| ``AIIDA_PATH = '~/project_b/.aiida'``                        | ``~/project_b/.aiida``           |
+| ``AIIDA_PATH = '~/project_a/.aiida:~/project_b/.aiida'``     | ``~/project_a/.aiida``           |
+| ``AIIDA_PATH = '~/project_a/subfolder:~/project_a/.aiida:'`` | ``~/project_b/.aiida``           |
```

Comment:
yep, good catch, that's a typo

Reply:
Actually, it should be
```
``AIIDA_PATH = '~/project_a/subfolder:~/project_a/.aiida:'`` | ``~/project_a/subfolder/.aiida``
```
At least that is the current behavior. Because `~/project_a/subfolder` contains an `.aiida` folder, that is the first hit and it is selected. The paths in `AIIDA_PATH` don't necessarily need to include the `.aiida` part explicitly

Reply:
Ah, yes, of course!